systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -158,7 +159,25 @@ void testBigAmountOfScramShaUsers(ExtensionContext extensionContext) {
         String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
         String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
 
-        createBigAmountOfUsers(extensionContext, userName, "SCRAM_SHA");
+        createBigAmountOfUsers(extensionContext, NAMESPACE, userClusterName, userName, "SCRAM_SHA", 100);
+    }
+
+    @Tag(SCALABILITY)
+    @ParallelNamespaceTest
+    void testAlterBigAmountOfScramShaUsers(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
+        int numberOfUsers = 100;
+        int producerRate = 1000;
+        int consumerRate = 1500;
+        int requestsPercentage = 42;
+        double mutationRate = 3.0;
+
+        deployKafka(extensionContext, clusterName);
```

Comment:
why do you need a second kafka?

Reply:
Trying to fix the tests - no more use of 2 kafkas. Moved to isolatedTests approach. Still seeing issue with UserOperator. Talking with our developers about this. There might be a bug in UO..